### DISCUSSION TITLE
is this gonna have a discord server

### DISCUSSION THREAD

**Discussion by daisukidaisy** [Leather128/LeathersFunkinEngine#71] [General]
tbh i just want to find a fnf modding server that isnt too massive to maintain an on-topic convo but still has a respectable number of people who actually know coding. i want to learn this crap myself so i can actually be of some help instead of throwing random ideas around with no clue about the logistics

**Comment by Leather128** [comment]
uh i mean the closest right now is probably the ronezkj15 discord server (has leather engine channels, and people talk regularly about it), while also not having too many members [server invite here](https://discord.com/invite/GKSDmh2DYS). maybe in the future i'll make a dedicated server for leather engine though
